picard/options.py

```diff
@@ -372,7 +372,10 @@ def make_default_toolbar_layout():
 BoolOption('setting', 'standardize_instruments', True, title=N_("Use standardized instrument credits"))
 BoolOption('setting', 'standardize_vocals', True, title=N_("Use standardized vocal credits"))
 BoolOption('setting', 'track_ars', False, title=N_("Use track and release relationships"))
+# Translation toggles
 BoolOption('setting', 'translate_artist_names', False, title=N_("Translate artist names"))
+BoolOption('setting', 'translate_album_names', False, title=N_("Translate album names"))
```

Comment:
I think this should be:

```suggestion
BoolOption('setting', 'translate_album_titles', False, title=N_("Translate album titles"))
```

Reply:
Yes, but MusicBrainz is using "title" (see https://musicbrainz.org/doc/Style/Release)

Reply:
They're not consistent then.  `Album Title` does not appear on that page, while `Album Name` does.

Edit: I think `title` in that context more refers to `Release Title` though?  Anyway, we're splitting hairs.  I'll make the change unless there's contrary opinion (i.e. opinion to keep `Album name`).

> They can be used as part of a title when the release has several mediums with the same name (for example, in a box set that includes a double album, it can make sense to title those two mediums "Album Name, Disc 1" and "Album Name, Disc 2").